src/mmrelay/migrate.py

```
@@ -0,0 +1,1898 @@
+                for db_path in selected_group:
+                    dest = new_db_dir / db_path.name
+                    if dest.exists():
+                        try:
+                            dest.unlink()
+                            logger.debug("Deleted copied file: %s", dest)
+                        except (OSError, IOError):
+                            logger.warning("Failed to delete copied file: %s", dest)
+                raise MigrationError.integrity_check_failed(result[0])
+            logger.info("Database integrity check passed")
+        except sqlite3.DatabaseError as e:
+            logger.error("Database verification failed: %s", e)
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)